templates/web/rutland/about/faq-en-gb.html

```diff
@@ -0,0 +1,175 @@
+            receive an initial email to acknowledge the report.
+        </p>
+        <p>
+            You will then receive an email every time that fault is updated. This could be when
+            a member of the public posts an update onto the fault or when we post a status update.
+        </p>
+        <p>
+            Road problems that are the responsibility of our Highways department, such as potholes,
+            will receive automatic updates at different stages. You will receive an email when the
+            problem has been inspected and another one when the fault is fixed.
+        </p>
+        <p>
+            Some faults may be marked as referred when they are not the responsibility of Rutland County Council.
+        </p>
+        <p>
+            All of these updates will also be posted on the mapped fault so that anyone can see the status of any
+            fault recorded by clicking on it on the map.
+        </p>
+    </dd>
+
+    <dt>How can I get an update on a fault?</dt>
+    <dd>
+        <p>
+            When you record a fault you can sign up to receive updates. You will then receive an initial
+            email to acknowledge the report. In time, you will receive a further email when the problem is
+            assigned for inspection, one if the problem needs to be referred to another department or
+            organisation, and a final email when it is fixed or requires no further work.
+        </p>
+        <p>
+            These updates will also be posted on the mapped fault so that anyone can see the status of
+            any fault recorded by clicking on it on the map.
+        </p>
+    </dd>
+
+    <dt>Can I report a fault on my mobile?</dt>
+    <dd>
+        <p>
+            Yes, you can view our website on your phone or tablet and it will automatically adapt to the
+            size of your screen.
+        </p>
+    </dd>
+</dl>
+
+<a name="practical"></a>
+<dl>
+    <dt>Inappropriate use</dt>
+    <dd>
+        Rutland County Council is not responsible for the content and
+        accuracy of material submitted by its users. Rutland County Council will monitor
+        this website for instances of inappropriate use, and reserves the right to moderate
+        comments if such instances occur.
+    </dd>
+
+    <dt>Why does the site use kilometres for measurements?</dt>
+    <dd>
+        The maps are from Ordnance Survey who based their grid reference
+        system on measurements in metres and kilometres.
+    </dd>
+
+    <dt>How do I report a fault outside of Rutland?</dt>
+    <dd>
+        The map only shows locations within Rutland to keep it locally focussed.
+    </dd>
+</dl>
+
+<a name="organisation"></a>
+<dl>
+    <dt>Acknowledgement</dt>
+    <dd>
+        This site was built by <a href="http://www.mysociety.org/">mySociety</a>,
+        in conjunction with the <a href="http://www.youngfoundation.org.uk/">Young Foundation</a>.
```

Comment:
Main FMS took this out ages ago, perhaps we need to go through and check them all :)